pasta_eln/installationTools.py

```diff
@@ -184,9 +184,10 @@ def installLinuxRoot(couchDBExists:bool, pathPasta:Path=Path(''), password:str='
       conf = createDefaultConfiguration('admin', password, pathPasta)
       with open(path,'w', encoding='utf-8') as fConf:
         fConf.write(json.dumps(conf, indent=2) )
+    cookie = ''.join(random.choice(string.ascii_letters+string.digits) for _ in range(42))
```

Comment:
If it's some random string, why cannot we just just use uuid?
https://docs.python.org/3/library/uuid.html